services/collection/collection.cpp

```diff
@@ -252,7 +252,7 @@ namespace services::collection {
                              session,
                              make_cursor(context_->resource(), operation_status_t::failure));
         } else {
-            auto result = make_cursor(context_->resource(), operation_status_t::failure);
+            auto result = make_cursor(context_->resource(), operation_status_t::success);
```

Comment:
Why you did this change? What motivation?

Reply:
There was a bug there, which i didn't catch in my previous update (#270). When selecting entries, if none were selected, cursor->is_success() should be false, and when something was selected it should become true, but it didn't

Reply:
Get it, but in future plz comment such cases in Pr.